introduction.md

```diff
@@ -5,3 +5,8 @@
 Hello my name is Kevin Abas, I am a student and the University of California of santa Cruz and am about to graduate. I need to sharpen my git skills before I enter the job market :)
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?